File: BookingPlaceAPI/BookingPlaceAPI/Models/UserRating.cs
namespace BookingPlaceAPI.Models
{
    public class UserRating
    {
        public int UserId { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public double Rate { get; set; }
        public string Text { get; set; }
        public string Phone { get; set; }
    }
}

File: AndroidApp/app/src/main/java/ru/nutscoon/mapsproject/Models/UserRating.java
package ru.nutscoon.mapsproject.Models;

public class UserRating {
    public int getUserId() {
        return userId;
    }

    public String getName() {
        return name;
    }

    public String getSurname() {
        return surname;
    }

    public double getRate() {
        return rate;
    }

    public String getText() {
        return text;
    }

    public String getPhone() {
        return phone;
    }

    private int userId;
    private String name;
    private String surname;
    private double rate;
    private String text;
    private String phone;
}
